crates/oxc_linter/src/rules/eslint/radix.rs

```diff
@@ -70,7 +70,7 @@ impl Rule for Radix {
             match call_expr.callee.without_parenthesized() {
                 Expression::Identifier(ident) => {
                     if ident.name == "parseInt"
-                        && ctx.symbols().get_symbol_id_from_name("parseInt").is_none()
+                        && ctx.symbols().is_global_reference(ident.reference_id().unwrap())
```

Comment:
Is there ever a case where `reference_id()` would return `None`?

Reply:
They are all `None` before semantic runs, and then all are always `Some` after semantic. Linter runs after semantic, so this `unwrap()` can never fail.

I would like to replace all these `Option<Id>`s with just plain `Id`s with 0 as a "sentinel value" equivalent to `None`, so we can avoid all this unwrapping (`ScopeId`, `SymbolId`, `ReferenceId`). But that's another issue...